packages/browser-wallet/src/popup/popupX/shared/IdCard/RejectedIdCard.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Rejected from '@assets/svgX/rejected.svg';
+import { identityProvidersAtom } from '@popup/store/identity';
+import { useAtomValue } from 'jotai';
+import { RejectedIdentity } from '@shared/storage/types';
+import { useTranslation } from 'react-i18next';
+import Text from '@popup/popupX/shared/Text';
+import IdCard from './IdCard';
+
+export type RejectedIdentityProps = {
+    /** Identity to show. */
+    identity: RejectedIdentity;
+};
+
+export default function RejectedIdCard({ identity }: RejectedIdentityProps) {
+    const { t } = useTranslation('x', { keyPrefix: 'sharedX' });
+    const providers = useAtomValue(identityProvidersAtom);
+    const provider = providers.find((p) => p.ipInfo.ipIdentity === identity.providerIndex);
+    const idProviderName = provider?.ipInfo.ipDescription.name ?? 'Unknown';
+    return (
+        <IdCard title={identity.name} subtitle={t('idCard.rejectedBy', { idProviderName })}>
+            <IdCard.Content className="rejected">
+                <Rejected />
```

Comment:
I don't know about the icon. It clearly states that there is an error, but I don't think it looks consistent when comparing with verified identities. IMO, removing the icon is better design-wise, but arguably less clear in terms of what's going on.

I would ask for design clarification here as well.

Personally, I would go for something like 
![Screenshot 2024-11-13 at 09 45 51](https://github.com/user-attachments/assets/0456ef61-7630-40f4-a492-6d113769fa9b)

But I don't think that looks fantastic either 🤷 

Also maybe it's redundant to state the rejection so many times as well? I think maybe remove it from the black part of the id card and just simply state the error detail received from the IDP?

Reply:
I agree the design could be improved, but I would suggest we iterate on it in the design document and stick to this for now.